v2/docs/res/benchmarking.md

```diff
@@ -20,12 +20,15 @@ following parameters.
 
 KVDB [parameters](../gs/params.md#kvdb-parameters):
 
-* `throttling.init_policy` with a value determined by `hse storage profile`
-* `durability.interval_ms` with a value appropriate for the application
-* `durability.mclass=staging` if a
-[staging media class](../gs/storage.md#media-classes) is configured for the KVDB
+* `throttling.init_policy` as determined by `hse storage profile`
+in the common case where the KVDB is configured with a capacity media class
```

Comment:
Can you reference cli.md, or write  "`hse storage profile`" HSE CLI command" ?

Reply:
Done.  Linked to example in cli.md.